commit ef336075cf932d7de6ca18f4db6abde9c75443a3
Author: Sage Weil <sweil@redhat.com>
Date:   Fri Feb 15 12:14:13 2019 -0600

    mgr/PyModuleRegistry: avoid null deref from get_progress_events
    
    active_modules may be null.
    
    Signed-off-by: Sage Weil <sage@redhat.com>

diff --git a/src/mgr/PyModuleRegistry.h b/src/mgr/PyModuleRegistry.h
index 743efc333f7..0e46af44e6e 100644
--- a/src/mgr/PyModuleRegistry.h
+++ b/src/mgr/PyModuleRegistry.h
@@ -151,7 +151,9 @@ public:
   void get_health_checks(health_check_map_t *checks);
 
   void get_progress_events(map<std::string,ProgressEvent> *events) {
-    return active_modules->get_progress_events(events);
+    if (active_modules) {
+      active_modules->get_progress_events(events);
+    }
   }
 
   // FIXME: breaking interface so that I don't have to go rewrite all
